core-services/image-mirroring/lvms/OWNERS

```diff
@@ -0,0 +1,15 @@
+approvers:
```

Comment:
Did you create this manually? If so, could you make it the same as in https://github.com/openshift/lvm-operator/blob/main/OWNERS?